### PR TITLE
fix(kibbeh): update Indonesian 1/x

### PR DESCRIPTION
**Pull request by s3ansh33p**
More changes to be added in this pr, i'll comment when I have done all of them

### CODE REVIEW

**Comment by s3ansh33p** [comment]
Should be ready to push

File: kibbeh/public/locales/id/translation.json [hunk 1]
```diff
@@ -35,12 +35,12 @@
     "_comment": "Respective Page UI Internationalization Strings",
     "admin": {
       "ban": "Blokir",
-      "userStaffandContrib": "User Staff & Contributions",
-      "staff": "Staff: ",
-      "contributions": "Contributions",
-      "username": "Username",
-      "usrStaff": "User Staff",
-      "usrContributions": "User Contributions",
+      "userStaffandContrib": "Kontrubsi pengguna & staf",
```
Comment (MeztliRA) [inline kibbeh/public/locales/id/translation.json:38]:
```suggestion
      "userStaffandContrib": "Kontribusi pengguna & staf",
```
fix the typo

File: kibbeh/public/locales/id/translation.json [hunk 2]
```diff
@@ -135,8 +135,8 @@
       "sendDM": "Kirim DM",
       "aboutSuffix": "",
       "errors": {
-        "blocked": "This user has blocked you.",
-        "default": "Whoops! We couldn't load this user."
+        "blocked": "Kamu diblokir oleh pengguna ini.",
+        "default": "Whoops! Kami tidak memuat pengguna ini."
```
Comment (MeztliRA) [inline kibbeh/public/locales/id/translation.json]:
```suggestion
        "default": "Whoops! Kami tidak bisa memuat pengguna ini."
```
make it more clear

File: kibbeh/public/locales/id/translation.json [hunk 3]
```diff
@@ -163,8 +163,8 @@
       "download_for": "Unduh untuk %platform% (%ext%)"
     },
     "privacySettings": {
-      "title": "Privacy Settings",
-      "header": "Privacy Settings",
+      "title": "Pengaturan privasi",
+      "header": "Ppengaturan privasi",
```
Comment (MeztliRA) [inline kibbeh/public/locales/id/translation.json]:
```suggestion
      "header": "Pengaturan privasi",
```
also fix this typo

**Pull request merged**